src/status_im/ui/screens/wallet/transactions/views.cljs

```diff
@@ -71,16 +72,71 @@
                       :icon-opts (merge styles/forward
                                         {:accessibility-label :show-transaction-button})}]]]])
 
+(defn etherscan-link [address]
+  (let [link @(re-frame/subscribe [:wallet/etherscan-link address])]
+    [react/touchable-highlight
+     {:on-press #(when link
+                   (.openURL react/linking link))}
```

Comment:
this should be moved to fx, and used in event

Reply:
that's debatable, one could argue that there is no need to involve the whole state for that, it doesn't actually need anything from the app state so having a local fn seems reasonable

Reply:
and also it is written this way in other parts of the app, should be fine